src/python/WMCore/Storage/Backends/AWSS3Impl.py

```diff
@@ -28,11 +29,13 @@ def createSourceName(self, protocol, pfn):
         """
         return "%s" % pfn
 
-    def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=None):
+    def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=None, authMethod=None, forceMethod=False):
         """
         _createStageOutCommand_
         Build an aws s3 copy command
         """
+        logging.info("Warning! AWSS3Impl does not support authMethod handling")
```

Comment:
How about changing this log level to warning (logging.warning()) in all plugins that are reporting such warning?

Reply:
done